### ISSUE TITLE
[BITV] 9.1.1.1a/3.7 - "Details" section / The combo box for selecting "Tags" - The button for deleting a tag does not have an alternative text. (2)

### ISSUE THREAD

**Issue by Pytal** [nextcloud/server#36718]
- [x] https://github.com/nextcloud/nextcloud-vue/pull/3968
- [x] https://github.com/nextcloud/server/pull/37065
- ~https://github.com/nextcloud/nextcloud-vue/pull/3946~

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-cd239865a5-v3-n7

![image](https://user-images.githubusercontent.com/24800714/218900669-91e67297-f385-4a45-b8e7-0d0628dd44d4.png)

</details>

- [x] 9.1.1.1a/10.7 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-cd239865a5-v10-n7

**Comment by Pytal** [comment]
Functionality additions to NcSelectTags being done in https://github.com/nextcloud/nextcloud-vue/pull/3946

**Comment by Pytal** [comment]
Options considered thus far are using NcSelectTags in https://github.com/nextcloud/server and passing prop overrides for systemtags functionality OR adding systemtags functionality to NcSelectTags itself as it already contains some systemtags specific code

Both of these options are infeasible in one way or another due to the discrepancy between internal and external state incurred by adding select, create, and delete functionality

For this reason as well as to maintain compatibility and to have an intuitive implementation we will need to make changes in https://github.com/nextcloud/nextcloud-vue and https://github.com/nextcloud/server

An opt-in prop will need to be added to override the custom internal state handling of NcSelectTags and all new systemtags specific logic will need to be done in the systemtags app

**Comment by Pytal** [comment]
`@nextcloud/vue` changes done in https://github.com/nextcloud/nextcloud-vue/pull/3968

Server changes done in https://github.com/nextcloud/server/pull/37065
